_posts/2018-07-31-how-to-move-to-https.md

```diff
@@ -0,0 +1,100 @@
+
+### Serve all resources over HTTPS: avoid mixed content
+
+Once you've installed your certificate, it's time to ensure all assets served from your pages are served over HTTPS. Naturally, this entire process should be completed in a staging environment before making the switch to your production environment. Completing a full transition to HTTPS requires attention to detail and diligence. "Mixed content", or serving assets from a page over both HTTP _and_ HTTPS, can be both tedious and insidious to rectify. The longer your site has been around, the more content there is and the more historic hands have been in the pot of content creation, the more work there will be to make the switch. Depending on your platform (CMS or otherwise) and how it was designed, there may be many avenues for different stakeholders to have included assets within a page over time. Developers, site admins, and content editors usually have the ability to add assets to a page. If any assets start with `http://`, they'll need to be updated to `https://` to prevent mixed content warnings.
+
+We have recently helped a client who has been publishing articles at a high cadence for over 10 years with many different stakeholders over that period. Practices weren't consistent and uncovering all the ways in which HTTP resources were served from the page was a substantial undertaking. Therefore, be prepared for a time investment here, there may be many areas to audit to ensure all assets from all your pages are being served over HTTPS. Some common ways to experience mixed content in which HTTP assets are served from a site whose HTML is served over HTTPS are:
+
+- Hard-coding a resource: e.g. `http://www.example.com/img/insecure-image.jpg`
+- Using a 3rd-party library or ad network reference: `http://www.example.com/js/analytics.js`
+  - This is common for libraries that haven't been updated in a while. Most all of them now serve the same assets over the same path securely with HTTPS.
+
+Even [practices that were previously encouraged by Paul Irish](https://www.paulirish.com/2010/the-protocol-relative-url/), a leading web architect for the Google Chrome team may have contributed to your mixed content problem, so don't feel bad. But, there will likely be work to be done.
+
```

Comment:
Missing comma - "Even practices that were previously encouraged by Paul Irish, a leading web architect for the Google Chrome team, may have contributed to your mixed content problem, so don't feel bad."

Reply:
I'd suggest changing "But, there will likely..." to "Just know there will likely be work to be done."